tools/pd-analysis/main.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"github.com/pingcap/log"
+	"github.com/pingcap/pd/tools/pd-analysis/analysis"
+	"go.uber.org/zap"
+	"os"
+)
+
+var (
+	input    = flag.String("input", "", "input pd log file, required.")
+	output   = flag.String("output", "", "output file, default output to stdout.")
+	logLevel = flag.String("logLevel", "info", "log level, default info.")
+	style    = flag.String("style", "", "analysis style, example: transfer-counter")
+	operator = flag.String("operator", "", "operator style, example: balance-region, balance-leader, transfer-hot-read-leader, move-hot-read-region, transfer-hot-write-leader, move-hot-write-region")
+	start    = flag.String("start", "", "start time, for example 2019/09/10 12:20:07, default: total file")
```

Comment:
It's better to keep consistent with others. BTW, I prefer to use e.g.

Reply:
I'm a little confused where does "keep consistent" refer to

Reply:
done